src/front/wgsl/mod.rs

```diff
@@ -175,6 +175,11 @@ impl<'a> Error<'a> {
                 labels: vec![(accessor_span.clone(), "invalid accessor".to_string())],
                 notes: vec![],
             },
+            Error::UnknownIdent(ref ident_span, ident) => ParseError {
+                message: format!("unknown identifier: '{}'", ident),
+                labels: vec![(ident_span.clone(), "unknown identifier".to_string())],
```

Comment:
I wonder if the second element in this tuple should just be `&'static str`

Reply:
I think we want to be able to use `String` there too, e.g. we have some `format!`s above

Reply:
there is quite a bit of repetition there between `message` and the second part of the label